docs(waf): Add comprehensive offline/air-gap installation guide for a… (#1623)
nginx/documentation: content/waf/install/disconnected-environment.md

When working with package files, you can install the packages directly in your disconnected environment, or add them to an internal repository.

The first step is to download the package files from your connected environment that has internet access and NGINX repository credentials.

> This will vary based on your operating system choice, which determines your package manager. Select your operating system below.

Reviewer: ```suggestion
See the section for your operating system below:
```
